apps/web/src/components/Onboarding/SetupPassword/SetupPassword.tsx

```diff
@@ -198,14 +198,16 @@ export const SetupPassword = ({ mode }: SetupPasswordProps) => {
           <ModalBody>
             <Flex flexDirection="column" gap="24px">
               <PasswordInput
+                checkPasswordStrength={!isPasswordSet}
                 inputName="password"
                 label={isPasswordSet ? "Password" : "Set Password"}
+                required="Password is required"
```

Comment:
Since there may be a situation where the password is required and the strength check is not needed, I think it's better to separate the required check from the rest of the scheme

Reply:
Thank you, makes sense 👍